homeassistant/components/lifx/config_flow.py

```diff
@@ -93,7 +93,7 @@ async def _async_handle_discovery(
                 host, serial=serial, raise_on_progress=True
             )
         ):
-            return self.async_abort(reason="cannot_connect")
+            return self.async_abort(reason="cannot_connect")  # pragma: no cover
```

Comment:
We aren't allowed to use no cover in config flow tests so we need to figure out a way to get this line covered

Reply:
I'll remove the no cover and leave it as is with these lines uncovered, which at least maintains the status quo. We can review the tests in another PR.

Reply:
Sounds good 👍